### PR TITLE
[LIEF-12163] Fix security vulnerability

### PR DESCRIPTION
**Pull request by irmela**

### CODE REVIEW

File: twilio-call-setup.gemspec [hunk 1]
```diff
@@ -19,11 +19,12 @@ Gem::Specification.new do |gem|
...
   gem.require_paths         = ["lib"]
 
-  gem.required_ruby_version = ">= 2.1"
+  gem.required_ruby_version = ">= 2.2"
 
   gem.add_dependency "twilio-ruby", "~> 5.10"
 
   gem.add_development_dependency "rspec"
   gem.add_development_dependency "rubocop"
+  gem.add_development_dependency "rubocop-performance"
```
Comment (RKushnir) [inline twilio-call-setup.gemspec]: Is this related?

**Comment by irmela** [comment]
```
Post-install message from rubocop:
Performance Cops will be removed from RuboCop 0.68. Use rubocop-performance gem instead.

Put this in your Gemfile.

  gem 'rubocop-performance'

And then execute:

  $ bundle install

Put this into your .rubocop.yml.

  require: rubocop-performance

More information: https://github.com/rubocop-hq/rubocop-performance
```

@RKushnir I just thought it won't hurt to already include `rubocop-performance`

**Comment by irmela** [comment]
`nokogiri-1.10.2 requires ruby version >= 2.3.0, which is incompatible with the`

**Comment by RKushnir** [comment]
:+1:
